mne/io/egi/tests/test_egi.py

```diff
@@ -29,6 +29,9 @@
 egi_pause_fname = op.join(egi_path, 'test_egi_multiepoch_paused.mff')
 egi_eprime_pause_fname = op.join(egi_path, 'test_egi_multiepoch_eprime.mff')
 egi_pause_w1337_fname = op.join(egi_path, 'w1337_20191014_105416.mff')
+egi_mff_evoked_fname = op.join(egi_path, 'test_egi_evoked.mff')
+egi_txt_evoked_cat1_fname = op.join(base_dir, 'test_egi_evoked_cat1.txt')
+egi_txt_evoked_cat2_fname = op.join(base_dir, 'test_egi_evoked_cat2.txt')
```

Comment:
These files should probably live in `mne-testing-data` as well. No reason to keep them in this repo when the relevant file that's used with them is in `mne-testing-data`

Reply:
Got it. I put them in `mne/io/egi/tests/data` originally because that is where the `test_egi.txt` signals live, but I can certainly move then to `mne-testing-data`.

Reply:
That one is paired with the `test_egi.raw` file that also lives in this repo. We've abandoned the practice of adding files to the repo for testing unless they are tiny and standalone -- this one is coupled to the `testing` dataset so no benefit to having it here